traitsui/wx/led_editor.py

```diff
@@ -59,18 +53,12 @@ def update_editor(self):
         """
         self.control.SetValue(self.str_value)
 
+    def _alignment_changed(self):
+        if self.control is not None:
+            self.control.SetAlignment(LEDStyles[self.alignment])
```

Comment:
Could we make such a change in a separate PR such that this PR is more focused about exposing the existing editor as first-class / built-in / whatever editor?

Furthermore I expect there being very limited use case to mutate the alignment after the editor is created - we can wait for this feature request to arise. The fewer mutables there are, the simpler and more maintainable the code is.